### DISCUSSION TITLE
Unable to verify contract

### DISCUSSION THREAD

**Discussion by wee4050** [hashlips-lab/nft-erc721-collection#329] [Q&A]
When I used the program to deploy the contract, I also checked that the API was correct, but the contract could not be verified, and the following error occurred.
![屏幕截图 2022-06-16 112509](https://user-images.githubusercontent.com/96134975/173985189-f6e39ada-82ec-470e-9fae-ed8dca712cae.png)

**Comment by progressaded** [comment]
I has the same issue  , I just ran the command "yarn deploy --network truffle" again  . This gave me a new contract adress and then I ran the verification with this new adress and it worked fine  .

Have a nice day adn I hope this helps
